### PR TITLE
Visitor can navigate to 'How This Works' and 'Contact' views

### PR DESCRIPTION
**Pull request by leiter007**
# [PT Story](https://www.pivotaltracker.com/story/show/166861570)

### Changes proposed in this PR
* Content and styling for the `How this works` component, containing instructions on how to use the app, including mock screenshots to be replaced later
* Adds route for `How this works` component in app.js
* Content and styling for the `Contact` component, containing contact info to BeingStockholm - provided by Yat
 
### What we have learned
* More on Semantic UI and site overrides

### Screenshots
Screenshot 2019-06-23 at 18 41 07
Screenshot 2019-06-23 at 18 40 48
Screenshot 2019-06-23 at 18 41 00

### CODE REVIEW

File: src/Components/Contact.jsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+                  </Grid.Column>
+                  <Grid.Column className="contact-text-columns">
+                    <p><b>Open Lab</b> Vallhallavägen 79, 114 28 Stockholm, Sweden</p>
+                  </Grid.Column>
+                </Grid.Row>
+
+              </Grid>
+
+              
+            </Container>
```
Comment (raptorf1) [inline src/Components/Contact.jsx:66]: delete empty lines

**Review by raptorf1 — CHANGES_REQUESTED** [review]

**Comment by zanenkn** [comment]
Id use some icons on the get in touch page instead of E, T and A. (yes, this is the graphic designer talking here lol. Also ETA makes me think of "estimated time of arrival" which is totally irrelevant here.)
http://www.zondicons.com/icons.html

File: src/Components/Contact.jsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+                <p>
+                  We are particularly interested to build up a network of partners who would like to make the Stockholm region as welcoming and inclusive as possible for its residents, such as the public sector, major employers of internationals and those working on social cohesion and integration.
+                </p>
+              </Container>
+              <br></br>
+
+              <Container className="views-ingress-container">
+                <p>
+                  Do email us or give us a call!
+              </p>
```
Comment (zanenkn) [inline src/Components/Contact.jsx]: code formatting

File: src/Components/HowThisWorks.jsx [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+              <Container>
+                <p className="screenshot-label">
+                  Once we have verified your photo, your photo will be live on the map to inspire other Stockholmers to try out!
+                </p>
+              </Container>
+              <br></br>
+
+              <Header className="views-second-header" as='h3'>
+                How we use your information
+            </Header>
```
Comment (zanenkn) [inline src/Components/HowThisWorks.jsx]: code formatting

File: src/Components/HowThisWorks.jsx [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+              <Container className="views-text-container">
+                <p>
+                  <b>Your personal information will not be displayed publicly or shared with any third parties.</b> It will be used for Urban Beings' data analysis purposes only so that we can have a better understanding of how new and settled Stockholmers use the city. This includes your profile photo, name and email address and further information on your profile, such as which municipality you live in, where you lived previously, how long you have lived in Stockholm, your gender and age group.
+                </p>
+                <p>
+                  By taking part in the project, you allow us to use your photos and captions to be displayed on the Being Stockholm map, website and other marketing purposes such as (but not limited to) social media, exhibitions and pitches to prospective partners.
+                </p>
+                <p>
+                  Thank you for helping us understand how different Stockholmers use the city region for work and play!
+              </p>
```
Comment (zanenkn) [inline src/Components/HowThisWorks.jsx]: code formatting

**Review by zanenkn — CHANGES_REQUESTED** [review]

File: src/Components/Contact.jsx (hunk 2 above)
Comment (raptorf1) [reply src/Components/Contact.jsx]: done

File: src/Components/HowThisWorks.jsx (hunk 4 above)
Comment (raptorf1) [reply src/Components/HowThisWorks.jsx]: done

File: src/Components/HowThisWorks.jsx (hunk 3 above)
Comment (raptorf1) [reply src/Components/HowThisWorks.jsx]: done

**Comment by raptorf1** [comment]
> Id use some icons on the get in touch page instead of E, T and A. (yes, this is the graphic designer talking here lol. Also ETA makes me think of "estimated time of arrival" which is totally irrelevant here.)
> http://www.zondicons.com/icons.html

Done ;)

**Review by zanenkn — APPROVED** [review]

**Pull request merged**